avocado/utils/pci.py

```diff
@@ -135,7 +141,8 @@ def get_slot_from_sysfs(full_pci_address):
 
     :return: slot of PCI address from sysfs.
     """
-    if not os.path.isfile('/sys/bus/pci/devices/%s/devspec' % full_pci_address):
+    if not os.path.isfile('/sys/bus/pci/devices/%s/devspec' %
+                          full_pci_address):
```

Comment:
Please separate this additional change to a different patch as it is not related to the original change. Also (it's a matter of taste) but I personally prefer putting the `%` on the same line with arguments as the argument belongs to `%`, anyway there are different styles and this is currently not enforced).